watershed-iOS/Watershed/Watershed/WPTaskViewController.m

```diff
@@ -37,30 +39,75 @@ - (void)didReceiveMemoryWarning {
     [super didReceiveMemoryWarning];
 }
 
+#pragma mark - Button Action Setups
+
 - (void)addFieldReportAction {
     WPAddFieldReportViewController *addFieldReportViewController = [[WPAddFieldReportViewController alloc] init];
     [[self navigationController] pushViewController:addFieldReportViewController animated:YES];
 }
 
+- (void)deleteTask {
+    UIAlertView *confirmDelete = [[UIAlertView alloc] initWithTitle:@"Confirm Deletion"
+                                                            message:@"Are you sure you want to delete this task?"
+                                                           delegate:self
+                                                  cancelButtonTitle:@"Cancel"
+                                                  otherButtonTitles:@"Yes",nil];
+    [confirmDelete show];
+}
+
+- (void)editTask {
+    WPEditTaskViewController *editTaskViewController = [[WPEditTaskViewController alloc] init];
+    [editTaskViewController setTask:self.task];
+//    [editTaskNavController.navigationBar setBackgroundColor:[UIColor whiteColor]];
+//    [editTaskNavController.navigationBar setBarTintColor:[UIColor whiteColor]];
+//    [editTaskNavController.navigationBar setTitleTextAttributes:@{NSForegroundColorAttributeName : [UIColor blackColor]}];
```

Comment:
Does the view still look fine without this styling? Also we should make sure our views remain compatible with iOS 7 (we had a problem with that before when we were releasing the beta)